From f14a3085091a8fbb9c054be5b282646efd97bba7 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Wed, 16 May 2012 09:43:56 +0530
Subject: [PATCH 2/6] ARM: OMAP2+: am335x-mux: add ecap2_in_pwm2_out string
 entry

On am335x-sk board, backlight is control through ecap 2.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/mach-omap2/mux33xx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-omap2/mux33xx.c b/arch/arm/mach-omap2/mux33xx.c
index 59e51e0..5749d20 100644
--- a/arch/arm/mach-omap2/mux33xx.c
+++ b/arch/arm/mach-omap2/mux33xx.c
@@ -340,7 +340,7 @@ static struct omap_mux __initdata am33xx_muxmodes[] = {
 		"mmc2_sdcd", NULL, NULL, "gpio3_16"),
 	_AM33XX_MUXENTRY(MCASP0_AHCLKR, 0,
 		"mcasp0_ahclkr", NULL, "mcasp0_axr2", "spi1_cs0",
-		NULL, NULL, NULL, "gpio3_17"),
+		"ecap2_in_pwm2_out", NULL, NULL, "gpio3_17"),
 	_AM33XX_MUXENTRY(MCASP0_ACLKR, 0,
 		"mcasp0_aclkr", NULL, "mcasp0_axr2", "mcasp1_aclkx",
 		"mmc0_sdwp", NULL, NULL, "gpio3_18"),
-- 
1.8.1.4

